public/app/features/commandPalette/values.ts

```diff
@@ -1,5 +1,6 @@
-export const SCOPES_PRIORITY = 8;
-export const RECENT_SCOPES_PRIORITY = 7;
+// Bumped to way higher value, to give it more weight when searching
+export const RECENT_SCOPES_PRIORITY = 50;
```

Comment:
Why the "way higher" value? Isn't this the reason why then exact matches are problematic? Wouldn't just higher than scopes_priority be enough?

Reply:
> Why the "way higher" value? Isn't this the reason why than exact matches are problematic? Wouldn't just higher than scopes_priority be enough?

I tried constructing it with the same keywords as the Scopes section, but it still doesn't seem like it wants to be above regular Scopes results. Could it be because the scopes section gets like 10 matches, and the Recent scopes one gets 2 or 3?